pkg/ddl/multi_schema_change.go

```diff
@@ -34,19 +38,52 @@ func (d *ddl) MultiSchemaChange(ctx sessionctx.Context, ti ast.Ident, info *mode
 	if err != nil {
 		return errors.Trace(err)
 	}
+	involveSchemaInfo := make([]model.InvolvingSchemaInfo, 0, len(info.AddForeignKeys)+1)
+	if len(info.AddForeignKeys) > 0 {
+		involveSchemaInfo = append(involveSchemaInfo, model.InvolvingSchemaInfo{
+			Database: schema.Name.L,
+			Table:    t.Meta().Name.L,
+		})
+
+		for _, subjob := range info.SubJobs {
+			if subjob.Type != model.ActionAddForeignKey {
+				continue
+			}
+
+			if len(subjob.Args) < 1 {
+				ddllogutil.DDLLogger().Warn(
+					"expected add foreign key subjob args length",
+					zap.Any("args", subjob.Args))
+				continue
+			}
+			fkInfo, ok := subjob.Args[0].(*model.FKInfo)
+			if !ok {
```

Comment:
How about using `intest` and panic?

Reply:
Not sure we have a upgrading integration test and will see the old version DDL jobs while `intest` is true.

Reply:
OK. Anyway we should always turn on `intest` for all integration tests.